src/main/java/at/hannibal2/skyhanni/features/rift/CruxTalismanDisplay.kt

```diff
@@ -0,0 +1,70 @@
+package at.hannibal2.skyhanni.features.rift
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.ItemUtils.getInternalName
+import at.hannibal2.skyhanni.utils.ItemUtils.getLore
+import at.hannibal2.skyhanni.utils.LorenzUtils.addAsSingletonList
+import at.hannibal2.skyhanni.utils.RenderUtils.renderStringsAndItems
+import net.minecraftforge.event.world.WorldEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class CruxTalismanDisplay {
+
+    private val config get() = SkyHanniMod.feature.rift
+    private val partialName = "CRUX_TALISMAN"
+    private var display = listOf<List<Any>>()
+    private val displayLine = mutableListOf<String>()
+
+    @SubscribeEvent
+    fun onRenderOverlay(event: GuiRenderEvent.GameOverlayRenderEvent) {
+        if (!isEnabled()) return
+        config.cruxTalismanPosition.renderStringsAndItems(
+                display,
+                extraSpace = 0,
+                posLabel = "Crux Talisman Display"
+        )
+
+    }
+
+    private fun update() {
+        display = drawDisplay()
+    }
+
+    private fun drawDisplay(): List<List<Any>> {
+        return buildList { if (displayLine.isNotEmpty()) displayLine.forEach { addAsSingletonList(it) } }
+    }
+
+    @SubscribeEvent
+    fun onTick(event: LorenzTickEvent) {
+        if (!isEnabled()) return
+        if (!event.isMod(20)) return
+        displayLine.clear()
+        var found = false
+        val inventoryStack = InventoryUtils.getItemsInOwnInventory()
+        for (stack in inventoryStack) {
+            val internalName = stack.getInternalName()
+            if (internalName.contains("CRUX_TALISMAN".toRegex())) {
```

Comment:
could this not just be:
if (internalName.contains(partialName)) {

Reply:
right, forgot to change that